roles/sonarqube/defaults/main.yml

```diff
@@ -2,3 +2,6 @@
 sonarqube_project_name: devsecops
 sonarqube_project_display: DevSecOps Common Resources
 sonarqube_password: '{{ lookup("password", tmp_dir + "/sonarqube.password chars=ascii_letters,digits") }}'
+
+rhsso_realm_name: openshift
+rhsso_project_name: rhsso
```

Comment:
Missing properties here:
```yaml
rhsso_app_label: rhsso
subdomain: '{{ full_cluster_name }}'
```

Reply:
Once this is fixed, good to merge 👍

Reply:
Fixed. It looks like when I was testing I was running this together with the rhsso role and I missed these variables because of that.